tensorboard/plugins/hparams/_keras.py

```diff
@@ -62,10 +62,14 @@ def __init__(self, writer, hparams, trial_id=None):
         summary_v2.hparams_pb(self._hparams, trial_id=self._trial_id)
         if writer is None:
             raise TypeError(
-                "writer must be a `SummaryWriter` or `str`, not None"
+                "writer must be a `SummaryWriter`, `str` or `PathLike`, not None"
```

Comment:
Just a nitpick here but it would be nice to keep the formatting consistent
```suggestion
                "writer must be a `SummaryWriter`, `str`, or `PathLike`, not None"
```